app/views/devise/two_factor_authentication_setup/index.html.slim

```diff
@@ -10,5 +10,6 @@ h1.heading = t('devise.two_factor_authentication.two_factor_setup')
   p#2fa-description = t('devise.two_factor_authentication.otp_setup')
   = f.input :phone, as: :tel, label: 'Phone Number', required: true,
       input_html: { 'aria-describedby' => '2fa-description' }
+  = f.input :phone_sms_enabled, as: :boolean, label: 'SMS Enabled?', input_html: { checked: true }
```

Comment:
`SMS Enabled?` implies that if the phone can receive SMS, we will always send an SMS to it. Is that what we're saying, or do we want to allow users to prefer receiving a phone call even if their phone is capable of receiving SMS? If the latter, then perhaps it might be clearer to use 2 checkboxes: `I prefer to receive SMS`, and `I prefer to receive a phone call` for example.